language/move-bytecode-verifier/src/verifier.rs

```diff
@@ -27,6 +40,10 @@ use move_binary_format::{
 /// minimize the code locations that need to be updated should a new checker
 /// is introduced.
 pub fn verify_module(module: &CompiledModule) -> VMResult<()> {
+    verify_module_with_config(module, &VerifierConfig::default())
+}
+
+pub fn verify_module_with_config(module: &CompiledModule, config: &VerifierConfig) -> VMResult<()> {
```

Comment:
Random nit, typically we stick these sort of contexts/configs before the module, as the arg is shared between the various calls. But this is fine and I don't really care